### ISSUE TITLE
SetAxis Error dialog?

### ISSUE THREAD

**Issue by cowboy** [Lasse-B/Sx2vJoy#1]
@Lasse-B, every once in a while, I get this alert dialog, and I have no idea why. When I click OK, the dialog is just replaced by another one. The only way to make it go away (and for Sx2vJoy to work again) is to re-launch the app. But after a while, the dialog pops up again.

![](http://i.gyazo.com/929f5357112f56cd36b351af71db251d.png)

It seems to be triggered when I touch the SpaceMouse wheel, but only very occasionally.

**Comment by Lasse-B** [comment]
I got this too, only seems to happen after Sx2vJoy has been running for several hours. Looks to be an error in the vJoy.ahk library as it seems to go away when I call the vJoy dll directly, but I haven't gotten around to finalizing the new build yet. Probably some time next week when things aren't so hectic.

PS:
Just had it shortly after booting up the computer. That's a first for me. Anyway, I removed all connections to the vJoy AHK library. Need a day or two to test it though, so I'll probably put up the new build some time Saturday evening (UTC time zone).
